modules/scripts/src/base/collections.ts

```diff
@@ -1,23 +1,18 @@
-/// <reference path="../../../../scripts/bin/components/components.d.ts" />
-/// <reference path="../../../../scripts/bin/components/modules.d.ts" />
+import {BoundConstraintValidator, Descriptor, Map} from "../../../../scripts/bin/app/modules.api";
```

Comment:
agreed and fixed.
also I have a lot of phase 2 thoughts (regarding the polyfills, the kisckstart, analytics and in general, re-usability),  but I don't see these issues as blockers.